verification/cbmc/sources/proof_allocators.c

```diff
@@ -55,12 +55,12 @@ void *bounded_calloc(size_t num, size_t size) {
     size_t required_bytes;
     __CPROVER_assume(aws_mul_size_checked(num, size, &required_bytes) == AWS_OP_SUCCESS);
     __CPROVER_assume(required_bytes <= MAX_MALLOC);
-    return calloc(num, size);
+    return (size == 0) ? NULL : calloc(num, size);
```

Comment:
The actual allocator goes farther: it has a fatal assert that size is not 0 
https://github.com/awslabs/aws-c-common/blob/master/source/allocator.c#L69

Reply:
So I could add an assumption in all allocators to consider `size != 0`.

Reply:
Yeah. Won't block this PR on it, but probably a good idea to assert that as an added check